packages/core/README.md

```diff
@@ -207,3 +207,47 @@ var pid = core.getState("pidToKill");
 
 process.kill(pid);
 ```
+
+#### OIDC Token
+
+You can use these methods to interact with the GitHub OIDC provider and get a JWT ID token which would help to get access token from third party cloud providers.
+
+**Method Name**: getIDToken()
+
+**Inputs**
+
+audience : optional
+
+**Outputs**
+
+A [JWT](https://jwt.io/) ID Token
+
+In action's `main.ts`:
+```js
+const core = require('@actions/core');
+async function getIDTokenAction(): Promise<void> {
+  
+   const audience = core.getInput('audience', {required: false})
+   const id_token = await core.getIDToken(audience)
+   core.setOutput('id_token', id_token)
```

Comment:
Is it good practice to set this as an output? I don't know, I think we should just be careful what we put as an example. @thboop @TingluoHuang

Reply:
I have removed the `setOutput`, instead have just put a comment saying it can further be used to fetch access token from any third party cloud providers. Does this make sense?